gcs/src/main/scala/org/broadinstitute/monster/storage/gcs/GcsApi.scala

```diff
@@ -127,10 +394,17 @@ object GcsApi {
     }
 
   /** Build the JSON API endpoint for an existing bucket/path in GCS. */
-  def baseGcsUri(bucket: String, path: String): Uri =
+  def baseGcsUri(bucket: String, path: String): Uri = {
     // NOTE: Calls to the `/` method cause the RHS to be URL-encoded.
     // This is the correct behavior in this case, but it can cause confusion.
     uri"https://www.googleapis.com/storage/v1/b/" / bucket / "o" / path
+  }
+
+  /** Build the JSON API endpoint for an upload to a GCS bucket. */
+  def baseGcsUploadUri(bucket: String, uploadType: String): Uri = {
+    val uri = uri"https://www.googleapis.com/upload/storage/v1/b/" / bucket / "o"
```

Comment:
Does this work?
```suggestion
    val uri = uri"https://www.googleapis.com/upload/storage/v1/b" / bucket / "o"
```
If so, can you change it here and above? The double-slash makes me nervous now that I notice it...